src/components/TemplatedCaseStudy/SecondTextSection.js

```diff
@@ -0,0 +1,75 @@
+import React, { Fragment } from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import { Margin } from 'styled-components-spacing'
+
+import { Row, Col } from '../grid'
+import { SectionTitleStyles, BodyPrimary } from '../Typography'
+
+const NonMobileRow = styled(Row)`
+  display: none;
+
+  ${breakpoint('smallTablet')`
+    display: flex;
+  `}
+`
+
+const MobileOnlyRow = styled(Row)`
+  display: flex;
+
+  ${breakpoint('smallTablet')`
+    display: none;
+  `}
+`
+
+const LargeStatsValue = styled(BodyPrimary)`
+  ${SectionTitleStyles}
+`
+
+const Stats = ({ stats }) => (
+  <Fragment>
+    {stats.map(stat => (
+      <Margin bottom={1} key={stat.id}>
+        <LargeStatsValue>{stat.value}</LargeStatsValue>
+        <BodyPrimary bold noPaddingTop>
+          {stat.label}
+        </BodyPrimary>
+      </Margin>
+    ))}
+  </Fragment>
+)
+
+const TextColumn = ({ textBelowVideo }) => (
+  <Col width={[1, 1, 1, 1, 9 / 12, 7 / 12]}>
+    {textBelowVideo.map((text, i) => (
+      <BodyPrimary key={i}>{text}</BodyPrimary>
+    ))}
+  </Col>
+)
+
+const SecondTextSection = ({ hasStats, stats, textBelowVideo }) => (
+  <Fragment>
+    {hasStats ? (
+      <Fragment>
+        <NonMobileRow spaced>
+          <Col width={[0, 0, 0, 0, 3 / 12]}>
```

Comment:
I think we don't need the `0`s, just a single `[3/12]` is enough since the whole row will disappear on mobile? The same applies to line 62

Reply:
Yep true :+1:

I used to have this as a NonMobileCol & didn't think about it when I changed it to the entire row that's shown hidden.

Good stuff

Reply:
cool!